homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -287,7 +287,7 @@ def entity_registry_enabled_default(self) -> bool:
         if self.info.primary_value.command_class == CommandClass.SENSOR_BINARY:
             # Legacy binary sensors are phased out (replaced by notification sensors)
             # Disable by default to not confuse users
-            if self.info.node.device_class.generic != "Binary Sensor":
+            if self.info.node.device_class.generic.key != 0x20:
```

Comment:
In a future PR, let's store such values in constants as part of the Z-Wave Server Python lib.